### DISCUSSION TITLE
Ability to skip changelog with `goreleaser build`?

### DISCUSSION THREAD

**Discussion by n8felton** [goreleaser/goreleaser#3507] [Q&A] [answered]
Hi there, just found goreleaser today while working on a Packer plugin, so I'm very new to the community and this tools capabilities.

I've been trying to build locally only, using the `goreleaser build` command line. I found that [the docs state](https://goreleaser.com/cmd/goreleaser_build/):

> The `goreleaser build` command is analogous to the go build command, in the sense it only builds binaries.

Despite this, I have found that the `changelog` section is still attempting to be processed
```
changelog:
  use: github-native
```

Running `goreleaser build` I get this error, which is expected because i'm not setting my GITHUB_TOKEN, but I'm also not attempting to actually update anything on Github yet.
```
build failed after 3s                    error=POST https://api.github.com/repos/<redacted>: 401 Bad credentials []
```

If I remove the changelog section from my config, `goreleaser build` completes as I would expect - the binaries are built and available in the `dist` directory.

I'm hung up on if this is intended behavior and/or the best way to achieve what I'm trying to do, which is replace `go build` with `goreleaser build` for quick local testing.

**Comment by caarlos0** [comment, marked as answer]
oh, that seems like a bug, I think. Will look into it later today. Thanks for reporting!

**Reply by caarlos0** [reply]
its a bug, will convert this into an issue. thanks for reporting
